lms/static/js/fixtures/edxnotes/edxnotes.html

```diff
@@ -1,7 +1,7 @@
 <section class="container">
   <div class="wrapper-student-notes">
     <div class="student-notes">
-
+        <main id="main" aria-label="Content" tabindex="-1">
```

Comment:
For this one, could you outdent the `main` tags to match the inner tags? It's very odd to read when an inner element is less indented than the containing ones.